ImageItem/tests/AsyncRenderChildren-specs.js

```diff
@@ -0,0 +1,20 @@
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
+
+import AsyncRenderChildren from '../AsyncRenderChildren';
+
+let data = {};
+const Component = (props) => {
+	data = props;
+	return <AsyncRenderChildren {...props} />;
+};
+
+describe('AsyncRenderChildren', () => {
+	test('should have a fallback content', () => {
+		render(<Component fallback={<div>Loading...</div>} index={1}>children</Component>);
```

Comment:
Please add one more tc that chaning index value


		rerender(
			<Component fallback={<div>Loading...</div>} index={2}>children</Component>
		);

 AsyncRenderChildren.js |   66.66 |    42.85 |      75 |   66.66 | 25-27,33-34      

Then, the 33-34 will be coverd.
The rest are possible, but I'm not sure about timers yet.